packages/react-scripts/template/src/routes/__tests__/app.component.spec.js

```diff
@@ -18,20 +19,21 @@ jest.mock('react-redux', () => ({
 
 jest.mock('use-react-router');
 
+jest.mock('../../theme/initializeFontFace');
+
+jest.mock('../../theme/theme', () => ({
+  color: {
+    white: '#fffffff',
+  },
+  font: {
+    primary: 'OpenSans',
```

Comment:
Maybe export `OpenSans` as const?

Reply:
It's just a mock here - this could be any string, it's not dependant on any implementation